### ISSUE TITLE
Ruby on Rails 7 support?

### ISSUE THREAD

**Issue by addisonmartin** [OpenGems/redis_web_manager#45]
Is there planned support for Rails 7? The currently released version requires rails (< 7, >= 5.2). 

What are the issues preventing Rails 7+ support? Can I assist with upgrading?

Much thanks for a great gem!

**Comment by DumasOlivier** [comment]
Hey @addisonmartin , thanks for your issue, the version has been updated and @BorisBresciani is gonna upgrade the gem version soonish. Closing your issue ✅

**Comment by addisonmartin** [comment]
Great, thank you. Can confirm master branch is currently working with Ruby 3.0.3, Rails 7.0.0, and the new importmap way of handling assets.

**Comment by BorisBresciani** [comment]
@addisonmartin Not yet https://github.com/OpenGems/redis_web_manager/runs/4755307530 

You can help if you want

**Comment by BorisBresciani** [comment]
Update: https://github.com/OpenGems/redis_web_manager/runs/4756558547

**Comment by addisonmartin** [comment]
You're right, I just checked the engine could mount and I could click around it quickly. I will take a look and attempt to provide a PR later this week when I have some extra time.

**Comment by abarrak** [comment]
Hi, 
Is there progress regarding the upgrade or is it delayed due incompatibility issues?

**Comment by n-rodriguez** [comment]
See https://github.com/OpenGems/redis_web_manager/pull/56

**Comment by n-rodriguez** [comment]
ping @BorisBresciani

**Comment by miharekar** [comment]
Hey, just wanted to use this gem, but couldn't because it doesn't support Rails 7.

What's the blocker here?

**Comment by n-rodriguez** [comment]
> What's the blocker here?

@bdarcet

**Comment by n-rodriguez** [comment]
ping @BorisBresciani

**Comment by n-rodriguez** [comment]
ping @DumasOlivier

**Comment by n-rodriguez** [comment]
ping @BorisBresciani

**Comment by n-rodriguez** [comment]
ping @DumasOlivier
